modules/javafx.controls/src/main/java/javafx/scene/control/Control.java

```diff
@@ -222,8 +222,20 @@ private static Class<?> loadClass(final String className, final Object instance)
      * {@code Skin}. Every {@code Skin} maintains a back reference to the
      * {@code Control} via the {@link Skin#getSkinnable()} method.
      * <p>
+     * To ensure a one-to-one relationship between a {@code Control} and its {@code Skin},
+     * setting the {@link #skinProperty() skin property} to a non-null {@code Skin} first checks
```

Comment:
This need not be a link, since it is already in the skin property. I recommend changing it to code style.

Reply:
Actually, it's better to remove the link and leave it as plain text.